src/Images.css

```diff
@@ -16,3 +16,24 @@
     --pf-c-table__action--PaddingTop: 0.5rem;
     --pf-c-table__action--PaddingBottom: 0.5rem;
 }
+
+#containers-images .pf-c-table.pf-m-compact .pf-c-table__toggle {
+    padding-left: 0px;
+}
+
+.images-grey-text {
+   color: var(--pf-global--Color--200);
+}
+
+.images-title {
```

Comment:
How about wrapping the title with the stats in a https://www.patternfly.org/v4/layouts/flex/#basic. 

That wil add autopadding as well.

Reply:
Thanks, that works nicely and that means I can remove all the CSS yay!